src/test/java/com/ericsson/ei/frontend/BackendInformationControllerTest.java

```diff
@@ -86,7 +86,7 @@ public void testAddInstance() throws Exception {
         mockMvc.perform(MockMvcRequestBuilders.post("/add-instances")
             .accept(MediaType.APPLICATION_JSON_VALUE)
```

Comment:
You can use real HTTP Request library also,,, depends how near real environment we want to test. Both are ok to use.